### PR TITLE
WRP-3058: Increase code coverage for Arc, ArcPicker, ArcSlider, TemperatureControl

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Increased code coverage

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3058

### CODE REVIEW

File: Arc/tests/Arc-specs.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		render(<Arc data-testid="arc" />);
+		const svgPath = screen.getByTestId('arc').children.item(0);
+
+		const expected = '#000000';
+
+		expect(svgPath).toHaveAttribute('stroke', expected);
+	});
+
+	test('should change path color when `color` prop is defined', () => {
+		render(<Arc data-testid="arc" color="#00FF00" />);
```
Comment (stanca-pop-lgp) [inline Arc/tests/Arc-specs.js:29]: Please sort props.

File: Arc/tests/Arc-specs.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
```
Comment (stanca-pop-lgp) [inline Arc/tests/Arc-specs.js:30]: Please remove empty line.

File: ArcSlider/tests/ArcSlider-specs.js [hunk 3]
```diff
@@ -0,0 +1,188 @@
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
+
+import ArcSlider from '../ArcSlider';
+import {valueToAngle, angleToValue} from '../utils';
```
Comment (stanca-pop-lgp) [inline ArcSlider/tests/ArcSlider-specs.js:5]: Please sort import.

File: TemperatureControl/tests/TemperatureControl-specs.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+			const expected = '59°F';
+
+			expect(temperatureControl).toHaveTextContent(expected);
+		});
+
+		test('should convert unit to `Celsius` with `ko-KR` locale', () => {
+			ilib.setLocale('ko-KR');
+			render(<TemperatureControl />);
+
```
Comment (stanca-pop-lgp) [inline TemperatureControl/tests/TemperatureControl-specs.js:93]: Please remove empty line.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]

**Pull request merged**